config/betamocks/services_config.yml

```diff
@@ -926,3 +926,20 @@
       :cache_multiple_responses:
         :uid_location: url
         :uid_locator:  '\/avs\/(.*)'
+
+  # Sign Up Service API
+- :name: "MAP SUS"
+  :base_uri: <%= "#{URI(Settings.mobile_application_platform.sign_up_service_url).host}:#{URI(Settings.mobile_application_platform.sign_up_service_url).port}" %>
+  :endpoints:
```

Comment:
For all of these, we're going to want to use the `cache_multiple_responses` functionality, with a regex for `uid_locator` that extracts the `icn` in the path.

This will create mockdata entries with file paths that look like `map/sus/status/<icn>.yml`. And then finally we need to make sure to add one for a default and call it `map/sus/status/default.yml`